backend/api/v1/audit.go

```diff
@@ -382,11 +411,21 @@ func isAuditMethod(method string) bool {
 		v1pb.DatabaseService_BatchUpdateDatabases_FullMethodName,
 		v1pb.ProjectService_SetIamPolicy_FullMethodName,
 		v1pb.SQLService_Export_FullMethodName,
-		v1pb.SQLService_Query_FullMethodName:
+		v1pb.SQLService_Query_FullMethodName,
+		v1pb.RiskService_CreateRisk_FullMethodName,
+		v1pb.RiskService_DeleteRisk_FullMethodName,
+		v1pb.RiskService_UpdateRisk_FullMethodName,
+		v1pb.SettingService_UpdateSetting_FullMethodName:
 		return true
+	case v1pb.IssueService_CreateIssue_FullMethodName:
```

Comment:
Instead of recording grant request issues, let's audit log system-generated policy updates.